GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs

```diff
@@ -167,6 +168,43 @@ public IEnumerable<Shape> Get(string entityId)
             });
         }
 
+        /// <summary>
+        /// Returns the entities for the given id's.
+        /// </summary>
+        /// <param name="entityIds"></param>
+        /// <returns></returns>
+        public IEnumerable<Shape> Get(List<string> entityIds)
+        {
+            if (entityIds.Count() == 0)
+            {
+                return new List<Shape>();
+            }
+            var sql = new StringBuilder("SELECT id, shape_pt_lat, shape_pt_lon, shape_pt_sequence, shape_dist_traveled FROM shape WHERE FEED_ID = :feed_id AND id = :id0");
+            var parameters = new List<NpgsqlParameter>();
+            parameters.Add(new NpgsqlParameter("feed_id", DbType.Int64));
+            parameters[0].Value = _id;
+            int i = 0;
+            foreach (var entityId in entityIds)
+            {
+                if (i > 0) sql.Append($" OR id = :id{i}");
```

Comment:
Hopefully we never hit an issue with the SQL command being too long. Just keep it in mind though, could become an issue if you query with 1000's of ids.

Reply:
I'll test it quickly